app/js/actions/AnswerActions.js

```diff
@@ -20,11 +20,10 @@ AnswerActions = {
 
   receiveAnswers: (data) => {
     AppDispatcher.dispatch({
-      actionType: ZhishiConstants.ANSWER_INDEX,
+      actionType: ZhishiConstants.ANSWER_INDX,
```

Comment:
ANSWER_INDEX vs ANSWER_INDX? I think the first one is easier to read/understand.